### PR TITLE
Move tts_service from example to configuration

### PR DESCRIPTION
**Pull request by FletcherAU**
## Proposed change

The note under the example provided provides information on a paramter that's mutually exclusive with the one included. This moves the note to the appropriate section of the configuration.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

Partially addresses #29831

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by FletcherAU** [comment]
Looking at this again, I think it may be appropriate to just remove the note completely as it's covered by the first part of that field.

**Comment by c0ffeeca7** [comment]
> Looking at this again, I think it may be appropriate to just remove the note completely as it's covered by the first part of that field.

agreed. Thank you for the catch, @FletcherAU

File: source/_integrations/notify.tts.markdown [hunk 1]
```diff
@@ -37,7 +35,7 @@ Please note that the `tts_service` parameter, must match the `service_name` defi
     required: exclusive
     type: string
   tts_service:
-    description: "The `service_name` of a TTS platform. Either use `entity_id` or `tts_service` to target a TTS platform."
+    description: "The `service_name` of a TTS platform. Either use `entity_id` or `tts_service` to target a TTS platform. This must match the `service_name` defined in the TTS integration."
```
Comment (c0ffeeca7) [inline source/_integrations/notify.tts.markdown]:
```suggestion
    description: "The `service_name` of a TTS platform. Either use `entity_id` or `tts_service` to target a TTS platform."
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @FletcherAU

**Pull request merged**